.github/workflows/node.js.yml

```diff
@@ -31,17 +31,11 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           flags: unit
-      - uses: actions/checkout@v4
-        with:
-          repository: xyj-3/sarif-js-sdk
-          ref: e23ca2d8dc6d70a7959a2098eba8ac332de907bb
-          path: sarif-js-sdk
-      - name: Install ESLint formatter
-        run: yarn add file:sarif-js-sdk/packages/eslint-formatter-sarif
-      - name: Run ESLint with formatter
-        run: yarn lint --format @microsoft/eslint-formatter-sarif --output-file eslint-results.sarif
-      - name: Upload ESLint result to GitHub
-        uses: github/codeql-action/upload-sarif@v3
+      - run: yarn add @microsoft/eslint-formatter-sarif --save-dev
```

Comment:
Since you are doing this, could you just add the package to devDependencies?

Reply:
Actually you know what, I think I should just remove the `--save-dev` part because I was just copying it from the usage section. You don't actually use the sarif formatter anywhere outside the workflow so I think it makes sense only to install it in the workflow.